website/layout/AutodocsLayout.js

```diff
@@ -121,17 +170,10 @@ var ComponentDoc = React.createClass({
             </div>
           );
         })}
-        {Object.keys(style.props).map((name) =>
-          <div className="prop" key={name}>
-            <h6 className="propTitle">
-              {name}
-              {' '}
-              {style.props[name].type && <span className="propType">
-                {renderType(style.props[name].type)}
-              </span>}
-            </h6>
-          </div>
-        )}
+        {Object.keys(style.props)
+          .sort(sortByPlatform.bind(null, style.props))
+          .map((name) => this.renderStylesheetProp(name, style.props[name]))
```

Comment:
Not sure either :P I'm going to leave it as it was but if we have a convention at fb, happy to follow it